### DISCUSSION TITLE
Time dependent electric field

### DISCUSSION THREAD

**Discussion by ephhuang** [epochpic/epoch#595] [Q&A]
Hi all,
is adding an electric field that changes with time possible? I ask because the "field block" in the user manual only shows that the external electric field is dependent on space.

**Comment by Status-Mirror** [comment]
Hi @ephhuang,

When you use the [fields block](https://epochpic.github.io/documentation/input_deck/input_deck_fields.html), you're only setting the initial value for the $\pmb{E}$ and $\pmb{B}$ fields. These will then evolve according to the EPOCH field solver.

As far as I'm aware, you cannot currently add external fields to the simulation once it has started.

Apologies,
Stuart

**Reply by ephhuang** [reply]
Thank you so much for the reply!!
with best regards,
Ping Han
